FCCee/IDEA/compact/IDEA_o1_v03/DriftChamber_o1_v02.xml

```diff
@@ -0,0 +1,189 @@
+  <!--     <constant name="DCH_outer_cyl_R_total"          value=" 2001 * mm "  /> -->
+  <!--     <constant name="DCH_half_length_total"          value=" 2250 * mm "  /> -->
+
+  <!-- Gas (active volume) geometry     -->
+  <constant name="DCH_gas_inner_cyl_R"          value=" 350 * mm "   />
+  <constant name="DCH_gas_outer_cyl_R"          value=" 2000 * mm "  />
+  <constant name="DCH_gas_Lhalf"                value=" 2000 * mm "  />
+
+  <!--    Vessel cylinder surrounds the gas cylinder in radius and z     -->
+  <!--    Inner wall and outer wall have different thickness-->
+  <constant name="DCH_vessel_thickness_innerR"           value=" DCH_gas_inner_cyl_R - DCH_inner_cyl_R_total"      />
+  <constant name="DCH_vessel_thickness_outerR"           value=" DCH_outer_cyl_R_total - DCH_gas_outer_cyl_R"      />
+  <!--    Wall that make up the base of the cylindrical vessel includes services-->
+  <!--    filling the space from z=2m to z=2.25m  -->
+  <constant name="DCH_vessel_disk_zmin"                  value=" DCH_gas_Lhalf"                                    />
+  <constant name="DCH_vessel_disk_zmax"                  value=" DCH_half_length_total"                            />
+
+
+
+  <!-- Position of guard wires   -->
+  <!--    _z0 := at z=0          -->
+  <!--    _zL2 := at z=DCH_Lhalf -->
+  <constant name="DCH_guard_inner_r_at_z0"         value=" DCH_gas_inner_cyl_R + 4*mm "    />
+  <constant name="DCH_guard_outer_r_at_zL2"        value=" DCH_gas_outer_cyl_R - 12.5*mm " />
+
+  <!-- Parametrization of number of cells ands layer/superlayer   -->
+  <constant name="DCH_ncell"                value=" 192 "  />
+  <constant name="DCH_ncell_increment"      value=" 48 "   />
+  <constant name="DCH_nsuperlayers"         value=" 14 "   />
+  <constant name="DCH_nlayersPerSuperlayer" value=" 8 "    />
+  <constant name="DCH_ncell_per_sector"     value=" 24 "   />
+
+  <!-- Alfa = twistangle/2   -->
+  <constant name="DCH_alpha"                value=" 15*deg "    />
+
+  <!--  Parameters of first layer  -->
+  <constant name="DCH_first_sense_r"        value=" DCH_guard_inner_r_at_z0 + 8*mm "        />
+  <constant name="DCH_first_width"          value=" 2*pi* DCH_first_sense_r / DCH_ncell"    />
+
+  <!--
+  Details about geometry of wires:
+    - guard wire:             50 um Al (core), 0.3 um Ag (coating)
+    - sense wire:             20 um W  (core), 0.3 um Au (coating)
+    - field wires top/bottom: 40 um Al (core), 0.3 um Ag (coating)
+    - field wire center:      50 um Al (core), 0.3 um Ag (coating)
+  -->
+  <!-- sense wire thickness (total)   -->
+  <constant name="DCH_SWire_thickness"           value="0.0203*mm"   />
+
+  <!-- Field Side (top/bottom) wire thickness (total)   -->
+  <constant name="DCH_FSideWire_thickness"       value="0.0403*mm"   />
+
+  <!-- Field Central wire thickness (total)   -->
+  <constant name="DCH_FCentralWire_thickness"    value="0.0503*mm"   />
+
+  </define>
+  <materials>
```

Comment:
Could you put the materials in the file `materials_o1_v02.xml` please? Materials for the previous DC implementation are already defined there, inside the beacons "start of materials for drift chamber" ... "end of material for drift chamber", let's also make sure there are no duplicates.